### PR TITLE
Makeflow send resources

### PR DESCRIPTION
**Pull request by btovar**
Use the logic of work queue for retrying tasks in makeflow.

@nkremerh: This should solve your issue. Please test comprehensively, even without resources.

### CODE REVIEW

File: makeflow/src/dag_node.c [hunk 1]
```diff
@@ -281,53 +280,18 @@ void dag_node_init_resources(struct dag_node *n)
...
 {
-	if( n->resources_needed->cores > -1 )
-		debug(D_MAKEFLOW_RUN, "cores:  %"PRId64".\n",      n->resources_needed->cores);
-	if( n->resources_needed->memory > -1 )
-		debug(D_MAKEFLOW_RUN, "memory:   %"PRId64" MB.\n", n->resources_needed->memory);
-	if( n->resources_needed->disk > -1 )
-		debug(D_MAKEFLOW_RUN, "disk:     %"PRId64" MB.\n", n->resources_needed->disk);
-	if( n->resources_needed->gpus > -1 )
-		debug(D_MAKEFLOW_RUN, "gpus:  %"PRId64".\n",       n->resources_needed->gpus);
+	const struct rmsummary *r = dag_node_dynamic_label(n);
```
Comment (nkremerh) [inline makeflow/src/dag_node.c]: _r_ is coming back as null here in my testing. Any guess as to why?

**Comment by btovar** [comment]
Probably the  max resources per category are not being set correctly anymore.

**Comment by btovar** [comment]
Mmm, no. The max values are set correctly.

**Comment by nkremerh** [comment]
Runs like a dream.

**Pull request merged**